src/utility/Image.js

```diff
@@ -0,0 +1,15 @@
+// function loadImage() {
+//   return new Promise((resolve, reject) => {
+//     const img = new Image();
+//     img.src = imgSrc;
+//     img.onload = function() {
+//       alert("Image ratio" + img.width / img.height);
+//       resolve({ width, height });
+//     };
+//     img.onerror = function(error) {
+//       reject(error);
+//     };
+//   });
+//   const imgSrc = "https://cdn.pixabay.com/photo/2015/03/26/09/47/sky-690293_1280.jpg";
+//   return { width: 3, height: 1 };
+// }
```

Comment:
Do we need this file, if we dont need you can delete it. Thanks!

Reply:
deleted the file